.editorconfig

```diff
@@ -0,0 +1,11 @@
+# http://editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
```

Comment:
BTW vscode does not support it. I need install plugin :)  http://editorconfig.org/#download